src/index.html

```diff
@@ -45,11 +45,46 @@
       </div>
     </footer>
 
-    <script
-      src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
-      integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
-      crossorigin="anonymous"
-    ></script>
-    <script src="./index.js"></script>
+<!-- modal -->
+    <div id="item-modal" class="modal">
+      <div class="card-container">
```

Comment:
I think we can create the modal dynamically from a JS module. That way we can access api data.